src/TheApplet.vue

```diff
@@ -175,6 +175,8 @@ const props = defineProps({
  * However, they won't be reactive to any changes they receive. Read more about that in MainView.vue.
  * Any code run at this level will only run once when the applet is loaded.
  */
+// Disabling the warning below because this a template literal, not props destructuring
+// eslint-disable-next-line vue/no-setup-props-destructure
```

Comment:
I'd appreciate a double check on this assumption

Reply:
> Your assumption is correct, but I don't get any linting warning when I delete these two newly added lines so they seem unnecessary to me.

Strange, I only added them because I was getting linting errors

Reply:
removing and re-running has no errors so I'm going to remove this